lib/spack/env/cc

```diff
@@ -142,6 +151,12 @@ if [[ -z $mode ]]; then
     done
 fi
 
+# Set up rpath variable according to language.
+eval rpath=\$SPACK_${comp}_RPATH_ARG
+if [[ $rpath != *%s* ]]; then
+    die "SPACK_${comp}_RPATH_ARG must contain '%s' for parameter substitution."
```

Comment:
Why do we need `%s` in the rpath arg? If the cc wrapper is the only place that uses it, why not just use `${rpath}$dep/lib` instead of `${rpath/\%s/$dep\/lib}`?